client/src/pages/NeedHelp.js

```diff
@@ -127,9 +130,26 @@ const NeedHelp = withRouter((props) => {
     setState({ ...updatedAnswers });
     if (key === "location") {
       localStorage.setItem("needHelpAnswers", JSON.stringify(updatedAnswers));
+      let query = {
+        objective: 2, // indexOf "Offering Help"
```

Comment:
Why the need to use indices? Maybe an constants object would provide better to clarity in future development?

```
const OBJECTIVES = {
  ALL: "All",
  REQUEST: "Requesting Help",
  OFFER: "Offering Help",
};
```

But I'm okay with leaving this to future refactoring if all is okay otherwise.

Reply:
I used the index for the query `?objective=1` so the end user doesn't change it manually, or just to make it less obvious and not plain text. Using a string should be fine.

Reply:
> so the end user doesn't change it manually, or just to make it less obvious and not plain text. 

I don't think that's a concern.. For these kinds of values, ideally we'd have some constants (like https://github.com/FightPandemics/FightPandemics/blob/431a96a3b8fcc8dce952ad132c5a962f1f40f85f/backend/lib/models/Post.js#L6) but which could be used in both backend & frontend.

We can tackle that with some future refactoring issue/PR. But for now using the string "request|offer" would be better than index I think. 

O